File: backend/server2/routes/authRoutes.go
package routes

import (
	"server2/controllers"

	"github.com/gin-gonic/gin"
)

func AuthRoutes(r *gin.Engine) {
    auth := r.Group("/api/server2/v1/auth")
    {
        auth.POST("/signup", controllers.SignUp)
        auth.POST("/login", controllers.LogIn)
    }
}

File: backend/server1/src/routes/authRoutes.ts
import express, { Router } from 'express';
const router: Router = express.Router();

import { imageFile } from '@/middlewares/multer';
import { logIn, signUp } from "@/controllers/auth";


router.post('/signup', imageFile, signUp);
router.post('/login', logIn);

export default router;
